client/src/views/Feed/Feed.vue

```diff
@@ -0,0 +1,241 @@
+            >
+              <i class='far fa-heart fa-lg'
+                :class='{"fas": post.hasBeenLiked}'>
+              </i>
+              <p class='likes'>{{post.likes}}</p>
+            </div>
+            <div class='comments-icon'>
+              <i class='far fa-comment-alt fa-md'></i>
+              <p>{{post.comments.length}}</p>
+            </div>
+          </div>
+          <div class='content'>
+          <p class='caption'
+              ref='caption'
+              :class='[post.caption.length > 40 ? "caption" : "caption2"]'><span>{{post.username}}</span> {{post.caption}}</p>
+        </div>
+      </div>
+      </div>
+        <div class='big-btn' @click='test'>
+              <i class='fas fa-2x fa-plus' ></i>
+        </div>
+    </div>
+ </div>
+</template>
+
+<script>
+import { mapGetters, mapActions } from 'vuex'
+export default {
+  name: 'Feed',
+  data () {
+    return {
+    }
+  },
+  methods: {
+    ...mapActions('posts', ['updateLikes']),
+
+    play: function (event) {
+      let currentVideo = event.target
+      if (currentVideo.paused) {
+        currentVideo.play()
+      } else {
+        currentVideo.pause()
+      }
+    },
+    test () {
+      this.$router.push('/addpost')
+    }
+  },
+  computed: {
+    ...mapGetters('posts', ['getPosts'])
+  }
+}
+</script>
+
+<style lang='scss'>
+.big-btn{
+width: 75px;
+    background: #87e0f5;
+    border-radius: 50%;
+    height: 75px;
+    position: fixed;
+    bottom: 8%;
+    right: 3%;
+    cursor: pointer;
+    i{
+          position: absolute;
+          color: #fff;
+    left: 25px;
+    top: 23px;
+    }
+}
+.feed-post {
+    padding-top: 50px;
+    height: fit-content;
+        padding: 6px 0px;
+}
+.heart-and-comments {
+    flex-direction: column;
+    position: absolute;
+    right: 3%;
+       top: 35%;
+    bottom: 65%;
+          cursor: pointer;
+    .heart i{
+    display: inline;
+        color: #DBE4E2;
+    }
+    .likes{
+      display: inline-block;
+    text-align: center;
+    font-size: 3.2vmin;
+    width: 7vmin;
+    margin: 5px 0;
+        font-weight: 100;
+            color: #DBE4E2;
+    }
+    .comments-icon {
+          display: inline-block;
+        font-size: large;
+        height: 25px;
+        cursor: pointer
+    }
+    .comments-icon i::before,
+    .heart i::before{
+   font-size: 4vmin;
+        color: #DBE4E2;
+    }
+    .comments-icon p{
+    display: inline-block;
+    vertical-align: bottom;
+    font-size: 3.2vmin;
+    width: 7vmin;
+    margin: 5px 0;
+    text-align: center;
+        color: #DBE4E2;
+    }
+    .comments-counter {
+        text-align: center;
+    }
+}
+.content .caption{
+  position: absolute;
+ bottom: 3%;
+ width: 90%;
+font-size: 0.85rem;
+word-wrap:break-word;
+        span {
+            font-weight: bold;
+        }
+}
+.content .caption2{
+  position: absolute;
+  width: 88%;
+ bottom: 4%;
+font-size: 0.85rem;
```

Comment:
Something is wrong with identation with this css;)

Reply:
now, indentation I hope looking better